### ISSUE TITLE
VIC upgrade 1.1 to 1.2  migrated containers not displayed after appliance is powered off

### ISSUE THREAD

**Issue by singhshwetaQE** [vmware/vic-product#617]
OVA Build : https://storage.googleapis.com/vic-product-ova-builds/vic-73d10421-dev.ova

**Steps:**
- Followed Upgrade steps 1.1 to 1.2
- Powered on new appliance 
- Ran upgrade script
- Log on to screen console
- Log on to Admiral UI
- Validated migrated data in Admiral UI - 2 containers and 1 VCH  from 1.1 was migrated successfully.
- Delete old appliance
-Power off new appliance/ shutdown guest OS
-Power on new appliance

**Actual Result:** 2 containers and 1 VCH from 1.1 is not displayed in new appliance after Power off new appliance/ shutdown guest OS
**Expected Result:** migrated data in Admiral UI should be available.

**Comment by gmuleshkov** [comment]
It looks like that the upgrade for admiral and the data transformations were successful because right after the upgrade script the entities were available in the new admiral.
I suppose it is something related to the volumes that are attached on start of the admiral container inside the ova. @andrewtchin @anchal-agrawal can you please take a look. Let me know if I can help with something.

**Comment by singhshwetaQE** [comment]
I retried to locate migrated VCH and containers in Admiral UI - logout and logged back in - Migrated VCH and containers still not displayed in Admiral UI. Then I cleared browser cache and logged again -finally I was able to see VCH and containers in Admiral UI. But from customer point of view it's not a good experience if they can't see changes in UI  and need to clear browser cache or manual refresh or  logout and logged back in.
